README.md

```diff
@@ -1,8 +1,25 @@
-# amiga-app-template
+# camera-streamer
 
-This repository is designed to streamline the creation of a new application deployable to the Amiga brain.
+This repository is an example of an app you can create with the [`amiga-app-template`](https://github.com/farm-ng/amiga-app-template).
 
-For the most up-to-date documentation on using this template repository, please refer to:
+## Try out this app
+
+This app can be cloned directly onto your amiga and used!
+
+Just follow the instructions about `ssh`'ing into your Amiga and cloning custom apps on:
+
+[amiga.farm-ng.com - **Developing Custom Applications**](https://amiga.farm-ng.com/docs/brain/brain-apps).
+
+## Follow the tutorial
+
+We have a tutorial available that goes through the development of this app available at:
+
+[amiga.farm-ng.com - **Camera Streamer Tutorial**](https://amiga.farm-ng.com/docs/tutorials/camera_streamer/camera-streamer-overview).
```

Comment:
this link seems to not be found

Reply:
This link is coming in https://github.com/farm-ng/amiga-dev-kit/pull/124